components/scream/src/share/atm_process/atmosphere_process.hpp

```diff
@@ -195,39 +193,34 @@ class AtmosphereProcess : public ekat::enable_shared_from_this<AtmosphereProcess
   }
 
   // Register required/computed fields in the field repo
-  virtual void register_fields (FieldRepository<Real>& field_repo) const = 0;
+  virtual void register_fields (const std::map<std::string,std::shared_ptr<FieldManager<Real>>>& field_mgrs) const = 0;
```

Comment:
Aha, here's where the comment should be updated for `register_fields`. Maybe add `override to the overridden methods, and then you can remove the comments there.

Reply:
I went ahead and updated the comment in all derived classes. Although, we should prob use `override` more often, to avoid shadowing bugs.